__tests__/http2/1.0/index.test.js

```diff
@@ -0,0 +1,41 @@
+import https2 from '../../../functions/http2/1.0';
+
+describe('HTTPS2 Step', () => {
+  test('Makes a succesfull http call.', async () => {
+    expect.assertions(1);
+
+    const request = {
```

Comment:
Would be better if request is extracted out of scope of this test function and properties that are not shared are added to it. But not blocking merging